rust-stonk/tests/test_stockfinder.rs

```diff
@@ -0,0 +1,12 @@
+extern crate rust_stonk;
```

Comment:
Integration tests have their own rule. Please see: https://doc.rust-lang.org/book/ch11-03-test-organization.html

Reply:
Will not add at current time